### ISSUE TITLE
Requesting 'lock workstation' command

### ISSUE THREAD

**Issue by ohhai-kthxbai** [sleevezipper/hass-workstation-service#48]
Hi 

In addition to:

ShutdownCommand
RestartCommand
LogOffCommand

Could you please add a LockWorkstationCommand

I think the command to instigate is:
`Rundll32.exe user32.dll,LockWorkStation`

Thank you!

**Comment by sleevezipper** [comment]
Yeah that would be pretty convenient! For now you can use a CustomCommand.
